### ISSUE TITLE
Keyboard shortcuts (Alt+s) conflict

### ISSUE THREAD

**Issue by MtMovo** [wireapp/wire-desktop#562]
Is there an option to change keyboard shortcuts binding?

The specific right Alt + s mutes the conversation which is unfortunate cause in my keyboard layout right Alt + 'key' produces special characters.

Thanks

**Comment by maximbaz** [comment]
Nope 😞 https://github.com/wireapp/wire-webapp/issues/94

**Comment by pkosiec** [comment]
I think a simple toggle in Settings will do in this case. It's really annoying - it makes the desktop app unusable for Polish users.

**Comment by pkosiec** [comment]
Why is this closed without any comment? Looks like issues stopping users from using the app don't matter at all...

**Comment by gregor** [comment]
The issue was fixed after all (https://github.com/wireapp/wire-webapp/pull/1397) and has been released to the public for a while.

**Comment by pkosiec** [comment]
Thanks for elaborating. Unfortunately this is just a half of the problem as "ń" character is still blocked by shortcut starting a new conversation - Alt + N - is that right or it has been fixed as well?

**Comment by gregor** [comment]
Our Polish native speakers inhouse did not have a problem with Alt-N to be honest. Can you please open a separate issue for this if it is still a problem which makes it easier to track for us as.

**Comment by pkosiec** [comment]
I'm a Mac user and it's occurring only on Windows machines, I've tested it about 5 months ago. If the shortcut is Ctrl + Alt + N, then the problem is still there.

**Comment by gregor** [comment]
Ok, thanks for letting me know.

**Comment by gregor** [comment]
Then it is fixed. We actually changed that combo as well for this specific scenario. For got about for a moment. https://github.com/wireapp/wire-webapp/pull/1529

**Comment by pkosiec** [comment]
Awesome, thank you!

**Comment by max-arnold** [comment]
I sent unintended pings multiple times by pressing Cmd+K in a wrong window, so I made this Hammerspoon script to disable the hotkey on a Mac:

```lua
wire_hotkeys = hs.hotkey.modal.new({}, nil)
wire_hotkeys:bind({"cmd"}, "k", nil, function() end)

function applicationWatcher(appName, eventType, appObject)
    if (eventType == hs.application.watcher.activated) then
        if (appName == "Wire") then
            wire_hotkeys:enter()
        end
    elseif (eventType == hs.application.watcher.deactivated) then
        if (appName == "Wire") then
            wire_hotkeys:exit()
        end
    end
end

appWatcher = hs.application.watcher.new(applicationWatcher)
appWatcher:start()
```

You can count this comment as another vote for customizable hotkeys.
